tools/cli/internal/openapi/filter/path.go

```diff
@@ -16,13 +16,80 @@ package filter
 import (
 	"log"
 
+	"github.com/mongodb/openapi/tools/cli/internal/apiversion"
+
 	"github.com/getkin/kin-openapi/openapi3"
 )
 
 type PathFilter struct {
 }
 
-func (f *PathFilter) Apply(doc *openapi3.T) error {
+func (f *PathFilter) Apply(doc *openapi3.T, metadata *Metadata) error {
 	log.Printf("Applying path for OAS with Title %s", doc.Info.Title)
+	for path, pathItem := range doc.Paths.Map() {
+		log.Printf("Path: %s", path)
+		filterPathItem(pathItem, metadata)
+	}
 	return nil
 }
+
+func filterPathItem(pPath *openapi3.PathItem, m *Metadata) {
+	version := m.targetVersion
+	for _, op := range pPath.Operations() {
+		latestMatchedVersion, err := getLatestVersionMatch(op, version)
+		if err != nil {
+			log.Fatalf("Error getting latest version match: %s", err)
+			return
+		}
+
+		log.Printf("Parsing OperationId: %s for targetVersion %s and got the latest matched version: %s",
+			op.OperationID, version, latestMatchedVersion)
+		// TODO: Continue parsing...
```

Comment:
Is this intentional?

Reply:
yep, not all functionality is there yet, it's a bunch of tickets to get there